be/src/vec/functions/function_date_or_datetime_computation.h

```diff
@@ -1093,6 +1093,27 @@ struct TimeToSecImpl {
     }
 };
 
+struct SecToTimeImpl {
+    using ReturnType = DataTypeTime;
+    static constexpr auto name = "sec_to_time";
+    static Status execute(FunctionContext* context, Block& block, const ColumnNumbers& arguments,
+                          size_t result, size_t input_rows_count) {
+        auto res_col = ColumnFloat64::create();
+        const auto& [argument_column, arg_const] =
```

Comment:
`use_default_implementation_for_constants ` is true, no need to dispose the const column